gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -126,6 +128,7 @@ def add_arguments(self, parser):
               'optimize_for_large_inputs are enabled.'))
 
   def validate(self, parsed_args, client=None):
+    # type: (argparse.Namespace) -> None
```

Comment:
I didn't find any function calls pass `client` except the unit tests. I wonder whether we can remove it.

Reply:
I think unit tests is the main use case for it actually. We should keep it. Could you add the type for it?